az-digital/az_quickstart
# Closes #1968 AZ News: enforce module dependency for its config files in order to allow reinstallation of az_news module, or a clean uninstallation.

## Description
In order to allow the az_news module to be uninstalled, and reinstalled without errors, I think we should enforce an az_news module dependency on all configuration provided by the az_news module.

This has the possibility of breaking any sites that have disabled az_news but continue to use the configuration provided by the module initially.

## Related issues
Closes #1968

## How to test

Uninstall az_news
Try to use the az_news content type
See that it has been deleted.
Reinstall az_news

## Types of changes

### Arizona Quickstart (install profile, custom modules, custom theme)
- Patch release changes
- Minor release changes

### Drupal core
- Patch release changes
- Minor release changes

### Drupal contrib projects
- Patch release changes
- Minor release changes

## Checklist
